HISTORY.md

```diff
@@ -1,5 +1,55 @@
 ### mlpack ?.?.?
 ###### ????-??-??
+  * Migrate from boost tests to Catch2 framework (#2523), (#2584).
+
+  * Update NumFOCUS team voting responsibilities (#3071).
+
+  * Add armadillo bump version policy (#3048).
+
+  * Bump minimum armadillo version from 8.400 to 9.800 (#3043).
+
+  * Adding a copy constructor in the Convolution layer (#3067).
+
+  * Replace boost::spirit parser by a local efficient implementation
+   (#2942).
+
+  * Disable correctly the autodownloader + fix tests stability (#3076).
+
+  * Replace boost::any with core::v2::any (#3006).
```

Comment:
```suggestion
  * Replace boost::any with core::v2::any or std::any if available
    (#3006).
```
I think that is correct?  Feel free to ignore the suggestion if it's not.

Reply:
Yes it is valid for Windows